hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1588,15 +1588,13 @@
       desc: '''Verify broadcast signals from lc_ctrl.
 
             - Preload the LC partition in the OTP ctrl with the following states: RMA, DEV,
-              TEST_LOCKED[N] & SCRAP (tested in chip_lc_ctrl_escalation).
+              TEST_LOCKED[N] & SCRAP.
             - Verify that the following broadcast signals are having the right effect in the
               respective IPs that consume them:
-              - lc_dft_en_o: impacts clkmgr, pinmux, OTP ctrl & AST
+              - lc_dft_en_o: impacts pinmux, OTP ctrl, flash ctrl, AST
```

Comment:
@msfschaffner how this impacts flash_ctrl?

Reply:
Ah sorry about that, this particular one does not affect flash_ctrl. I removed it.

However, `lc_nvm_debug_en_o` further below does impact it. 
Do you happen to know whether we test this at the system level?

Reply:
> Ah sorry about that, this particular one does not affect flash_ctrl. I removed it.
> 
> However, `lc_nvm_debug_en_o` further below does impact it. Do you happen to know whether we test this at the system level?

We have two relevant tests. Let me take a look and confirm what these tests are.

Reply:
Opened an issue here: #15273